apps/glusterfs/app_volume.go

```diff
@@ -246,8 +246,22 @@ func (a *App) VolumeDelete(w http.ResponseWriter, r *http.Request) {
 			return err
 		}
 
-		return nil
+		if !volume.Info.Block {
```

Comment:
The details are currenlty in the other repo, because I wanted to have a single document describing it all and wanted to avoid duplication of document and effort. We can rip the heketi part out of the other repo and put it here, but I somehow had hoped that it would not be asking too much to look at the gluster-kubernetes repo which is associated with heketi anyways. ;-)